biofefi/services/fi.py

```diff
@@ -78,3 +92,66 @@ def calculate_local_shap_values(
 
     # Return the DataFrame
     return shap_df, shap_values
+
+
+def load_fi_options(path: Path) -> dict:
+    """Load feature importance options.
+
+    Args:
+        path (Path): The path to the feature importance options file.
+
+    Returns:
+        dict: The feature importance options.
+    """
+
+    try:
+        with open(path, "r") as file:
+            fi_options = json.load(file)
+    except FileNotFoundError:
+        fi_options = None
+
+    return fi_options
+
+
+def find_previous_fi_results(experiment_path: Path) -> tuple[dict, dict]:
+    """Find previous feature importance results.
+
+    Args:
+        experiment_path (Path): The path to the experiment.
+
+    Returns:
+        tuple[dict, dict]: The feature importance options and results.
+    """
+    try:
+        fi_options = load_fi_options(fi_options_path(experiment_path))
+
+    except Exception:
+        fi_options = None
+
+    return fi_options
```

Comment:
Is this function meant to be about checking if FI results exist, could you check for the existence of the FI and Fuzzy options files, as will as the log dir for FI and Fuzzy, the plot dir for FI and Fuzzy and the results dir for them, too. Then make this function return a `bool`. Basically check if any of the paths in the function below exist and return `True` if so

Reply:
I also see this as more to do with experiments more broadly. The `services.fi` module is more about doing the actual FI calculations etc. So can you move this to `services.experiments`